Makefile

```diff
@@ -7,7 +7,7 @@ endif
 
 # Note: these targets should be run from the kuma vm
 django-tests: in_vagrant
-	python manage.py test
+	DJANGO_SETTINGS_MODULE=kuma.settings.testing python manage.py test
```

Comment:
Does anyone use this make rule? I ask because it's not mentioned in the docs anywhere and it's kind of awful--no one should ever run the tests this way without all the doohickey flags.